Character/Classes/Bard/Bard (Whispers).xml

```diff
@@ -1,52 +1,51 @@
+				<text>	Once you capture a shadow with this feature, you can't capture another one with it until you finish a short or long rest.</text>
+			</feature>
 		</autolevel>
 
 		<autolevel level="14">
-            <feature optional="YES">
-                <name>College of Whispers: Shadow Lore</name>
-                <text>At 14th level, you gain the ability to weave dark magic into your words and tap into a creature's deepest fears. </text>
-                <text>	As an action, you magically whisper a phrase that only one creature of your choice within 30 feet of you can hear. The target must make a Wisdom saving throw against your spell save DC. It automatically succeeds if it doesn't share a language with you or if it can't hear you. On a successful saving throw, your whisper sounds like unintelligible mumbling and has no effect. </text>
-                <text>	If the target fails its saving throw, it is charmed by you for the next 8 hours or until you or your allies attack or damage it. It interprets the whispers as a description of its most mortifying secret. While you gain no knowledge of this secret, the target is convinced you know it. </text>
-                <text>	While charmed in this way, the creature obeys your commands for fear that you will reveal its secret. It won't risk its life for you or fight for you, unless it was already inclined to do so. It grants you favors and gifts it would offer to a close friend. </text>
-                <text>	When the effect ends, the creature has no understanding of why it held you in such fear. Once you use this feature, you can't use it again until you finish a long rest.</text>
-            </feature>
+			<feature optional="YES">
+				<name>College of Whispers: Shadow Lore</name>
+				<text>At 14th level, you gain the ability to weave dark magic into your words and tap into a creature’s deepest fears.</text>
+				<text>	As an action, you magically whisper a phrase that only one creature of your choice within 30 feet of you can hear. The target must make a Wisdom saving throw against your spell save DC. It automatically succeeds if it doesn’t share a language with you or if it can’t hear you. On a successful saving throw, your whisper sounds like unintelligible mumbling and has no effect.</text>
+				<text>	If the target fails its saving throw, it is charmed by you for the next 8 hours or until you or your allies attack or damage it. It interprets the whispers as a description of its most mortifying secret.</text>
+				<text>	While you gain no knowledge of this secret, the target is convinced you know it. While charmed in this way, the creature obeys your commands for fear that you will reveal its secret. It won’t risk its life for you or fight for you, unless it was already inclined to do so. It grants you favors and gifts it would offer to a close friend.</text>
```

Comment:
```suggestion
				<text>	While you gain no knowledge of this secret, the target is convinced you know it. While charmed in this way, the creature obeys your commands for fear that you will reveal its secret. It won't risk its life for you or fight for you, unless it was already inclined to do so. It grants you favors and gifts it would offer to a close friend.</text>
```